From: Jakob Haufe <sur5r@sur5r.net>
Date: Mon, 9 Apr 2012 18:40:04 +0200
Subject: Ignore license.page
Forwarded: not-needed

---
 help/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

Index: git/help/Makefile.am
===================================================================
--- git.orig/help/Makefile.am	2016-01-23 13:53:12.806496131 +0100
+++ git/help/Makefile.am	2016-01-23 13:53:12.802496083 +0100
@@ -32,7 +32,6 @@
 	editprop.page		\
 	index.page		\
 	labelprop.page		\
-	license.page		\
 	mancreate.page		\
 	merge-ex1.page		\
 	merge-ex2.page		\
